server/src/main/java/org/opensearch/index/fielddata/fieldcomparator/BytesRefFieldComparatorSource.java

```diff
@@ -99,7 +100,7 @@ public FieldComparator<?> newComparator(String fieldname, int numHits, boolean e
         final boolean sortMissingLast = sortMissingLast(missingValue) ^ reversed;
         final BytesRef missingBytes = (BytesRef) missingObject(missingValue, reversed);
         if (indexFieldData instanceof IndexOrdinalsFieldData) {
-            return new FieldComparator.TermOrdValComparator(numHits, null, sortMissingLast) {
+            FieldComparator<?> cmp = new TermOrdValComparator(numHits, indexFieldData.getFieldName(), sortMissingLast, reversed) {
```

Comment:
@nknize  This may have caused a regression, see https://build.ci.opensearch.org/job/gradle-check/1941/

Reply:
(Also in https://build.ci.opensearch.org/job/gradle-check/1752/ associated with this PR)

Reply:
Disregard the above. Found the problem elsewhere.